client/src/styles/legacy/_style.scss

```diff
@@ -687,6 +687,25 @@ html, body {
   }
 }
 
+.validationerror .panel--padded .ui-tabs-nav {
+  margin-top: -$panel-padding-y / 2;
+
+  .nav-item {
+    display: flex;
+  }
+
+  .nav-link {
+    align-self: flex-end;
+  }
+
+  .tab-attention {
+    color: $state-draft;
+    align-self: flex-start;
+    margin-top: 12px;
+    margin-left: -4px;
```

Comment:
Using flex was actually something Sacha suggested to increase the vertical height of the tabs so that the white space between the error message and tab label OR tab icon stayed consistent

Normal height - no icon present

![image](https://user-images.githubusercontent.com/4809037/93954479-eadaef00-fda1-11ea-810e-c14683912f81.png)

Increased height - icon present

![image](https://user-images.githubusercontent.com/4809037/93954493-f5958400-fda1-11ea-8b7e-0537d82de86a.png)

Reply:
I think that's actually intentional, otherwise you end up with not enough whitespace between tabs

![image](https://user-images.githubusercontent.com/4809037/93955648-56be5700-fda4-11ea-8a6e-1c6c63194388.png)

@sachajudd ?

Reply:
I peer reviewed this yesterday with @Cheddam and we would like to keep the spacing between tabs consistent. I didn't notice this in our initial design review together.

Reply:
Updated